crates/frontend/src/styles/tab.rs

```diff
@@ -3,20 +3,15 @@ use futures_signals::signal::{self, Signal, SignalExt};
 use web_sys::HtmlElement;
 
 pub fn bar(dom: DomBuilder<HtmlElement>) -> DomBuilder<HtmlElement> {
-    dom.class("inline-flex")
-        .class("h-[35px]")
-        .class("gap-0")
-        .class("bg-lightgray")
+    dom.class("gap-0")
         .style("background-color", super::BACKGROUND_COLOR)
 }
 
 pub fn body(dom: DomBuilder<HtmlElement>,
     is_active: impl Signal<Item = bool> + 'static,
     mouse_over: impl Signal<Item = bool> + 'static
 ) -> DomBuilder<HtmlElement> {
-    dom.class("block")
-        .class("h-full")
-        .class("pt-1.5")
+    dom.class("pt-1.5")
```

Comment:
Super unimportant, but it would be more accurate to call this parameter `builder` (or `dom_builder`) instead of `dom`. Then usually the first call in the chain would go on the second line.
```rust
builder
    .class("pt-1.5")
    .class("pl-1")
```